hypha/apply/activity/models.py

```diff
@@ -41,6 +41,13 @@
 
 class BaseActivityQuerySet(models.QuerySet):
     def visible_to(self, user):
+        # To hide reviews from the applicant's activity feed
```

Comment:
Add a TODO about this is just for historic data and not needed for new data efter this.